phoenix-core/src/main/java/org/apache/phoenix/exception/SQLExceptionCode.java

```diff
@@ -480,8 +480,11 @@ public SQLException newException(SQLExceptionInfo info) {
                     "table with ROW_TIMESTAMP column."),
     CANNOT_UPSERT_WITH_SCN_FOR_MUTABLE_TABLE_WITH_INDEXES(903,"43M14",
             "Cannot use a connection with SCN set to " +
-                    "upsert data for a mutable table with indexes.");
+                    "upsert data for a mutable table with indexes."),
 
+    UNIMPLEMENTED_CASCADE_FEATURE(904,"43M15","This feature is WIP. Please refer to PHOENIX-5261"),
+
+    NOT_SUPPORTED_CASCADE_FEATURE(905,"43M16","CASCADE INDEX is not supported to alter view");
```

Comment:
QQ: What does 43M15/M16 mean?

Reply:
My only understanding is -- it is used as code for the exception, similar to standard SQL exception codes. I made sure it gets a unique code.